src/services/context_menu/context_menu_service.hpp

```diff
@@ -56,6 +56,10 @@ namespace big
 
 		Entity m_handle;
 		rage::fwEntity* m_pointer{};
+		rage::fwEntity* m_veh_pointer{};
+		rage::fwEntity* m_ped_pointer{};
+		rage::fwEntity* m_prop_pointer{};
```

Comment:
Can be removed after the proposed changes to `get_closest_to_center` have been implemented.